include/deal.II/lac/trilinos_precondition.h

```diff
@@ -1497,11 +1497,11 @@ namespace TrilinosWrappers
        * <tt>n_component</tt>, the provided @p constant_modes should fulfill
        * the following requirements:
        * <ul>
-       * <li>  n_component.size() == <tt>n_component</tt> </li>
-       * <li>  n_component[*].size() == n_dof_local or n_component[*].size()
-       * == n_dof_global </li>
-       * <li>  n_component[<tt>ic</tt>][<tt>id</tt>] ==
-       * "<tt>id</tt><em>th</em> DoF is corresponding to component <tt>ic</tt>
+       * <li>  <tt>constant_modes.size() == n_component</tt> </li>
+       * <li>  <tt>constant_modes[*].size() == n_dof_local</tt> or
+       *       <tt>constant_modes[*].size() == n_dof_global</tt> </li>
```

Comment:
Do you happen to know what `n_dof_local/global` actually is? Maybe it's better to use words than symbols here, along the lines of (not sure if this is actually correct, but let's pretend at least);
```
  <tt>constant_modes[*].size()</tt> needs to be equal to either the total number of degrees of freedom associated with the linear system, or the number of locally owned degrees of freedom. In parallel computations, the latter is the more appropriate choice since one does not want to store vectors of global size on individual processes.
```

Reply:
Yes, now I have checked it with a small piece of code and it is exactly as you write.